distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/cli/InstallPluginCommand.java

```diff
@@ -81,7 +95,55 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
             .collect(Collectors.toList());
         final boolean isBatch = options.has(batchOption);
 
+        if (plugins.isEmpty()) {
+            throw new UserException(ExitCodes.USAGE, "at least one plugin ID is required");
+        }
+
         InstallPluginAction action = new InstallPluginAction(terminal, env, isBatch);
-        action.execute(plugins);
+        try {
+            action.execute(plugins);
+        } catch (InstallPluginException e) {
+            int exitCode;
+
+            switch (e.getProblem()) {
+                case DUPLICATE_PLUGIN_ID:
+                case UNKNOWN_PLUGIN:
+                default:
```

Comment:
If we decide my above suggestion isn't worth while can we at least place the `default` path at the bottom. When we move to Java 17 this would be a great use case for switch expressions as the compiler would error if we failed to handle every scenario, since that's not yet the case I think the suggestion above is better as it _forces_ us to declare the right exit code for each condition.